src/pages/task/logs/LogMessageLine.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { LogMessage } from "gql/queries/get-task-logs";
+import { getLogLineWrapper } from "pages/task/logs/logMessageLine/LogLines";
+import { format } from "date-fns";
+const FORMAT_STR = "yyyy/mm/d, HH:mm:ss.SSS";
+
+export const LogMessageLine: React.FC<LogMessage> = props => {
+  const { timestamp, severity, message } = props;
+  const time = timestamp ? `[${format(new Date(timestamp), FORMAT_STR)}] ` : "";
```

Comment:
Looks like date-fns requires us to specifically import the locales we want, and then pass them into the format function. How are we planning on figuring that out? Feel free to defer this until later if you don't want to deal with it now